data/human/sales.txt

```diff
@@ -145,6 +145,55 @@ shipyard "Luxury Ships"
 	"Star Queen"
 	"Flivver"
 
+# For a few decades, New Tortuga has had advanced technology, Syndicate defectors, and a steady supply of poached employees. One of their top priorities has been technological advancement. They are only one planet though, so even with help from an advanced species, their success has been limited. Thus, this list is restricted low-to-mid-range tech from older companies. This is with the exception of the Syndicate, for which they already had access to top technology.
+shipyard "Independent (Korath-Allied)"
+	# Advanced Southern Pirates without the Osprey or Falcon:
+	"Modified Argosy"
+
+	# Commonly available from multiple sellers:
+	"Hauler"
+
+	# Common pirate fighter:
+	"Lance"
+
+	# Syndicate Advanced without the Container Transport or Arrow:
+	"Bulk Freighter"
+
+	# Non-modified versions of pirate-modified ships:
+	"Blackbird"
+	"Bastion"
+	"Behemoth"
+	"Bounder"
+
+	# Pirate modifications of standard ships:
+	"Marauder Bounder"
+	"Nighthawk (New Tortuga)"
+	"Cutthroat"
+
+
+# See note above about capabilities and limitations of "Independent (Korath-Allied)"
```

Comment:
I think the shipyard needs to be condensed, having many single entries with a comment explaining each is extremely cumbersome.  This suggestion reorganises the shipyard based on type.  It removes the Marauder Bounder, as I don't think it's appropriate to be offering Marauder ships for sale.  The Dagger has been added as @TheGiraffe3 suggested.  Leaving this open for comment rather than committing immediately.

```suggestion
shipyard "Independent (Korath-Allied)"
	"Blackbird"
	"Bounder"
	"Nighthawk (New Tortuga)"

	"Hauler"
	"Behemoth"
	"Bulk Freighter"

	"Cutthroat"
	"Modified Argosy"
	"Bastion"

	"Dagger"
	"Lance"


```

Reply:
I must admit that I do like knowing where they came from and how the list was created... But I also agree that this doesn't necessarily need to be preserved permanently. Having this be more visible and explained in-game would be preferred.